coding/python.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: base
+---
+
+# Python code standards
+
+This document outlines the rules for writing Python code across all our Projects.
+
+## Code style and formatting
+
+All Code should be written in accordance with [PEP 8](https://pep8.org/).
+To correctly format all your code please use [black](https://github.com/ambv/black). Since it differs in a small way from the standard **PEP8** make sure to pass `--line-length 79` when using it. The reason for black to do this is outlined [here](https://github.com/ambv/black#line-length).
```

Comment:
I don't think we need to explain Black's choices here - it seems a bit off-topic. People can do that research themselves if they really want to.

We might want to add a note that CI should reject any pull requests that don't conform to these standard though.